src/test/groovy/org/gradle/profiler/AndroidStudioIntegrationTest.groovy

```diff
@@ -0,0 +1,42 @@
+package org.gradle.profiler
+
+import org.gradle.profiler.studio.tools.StudioFinder
+import spock.lang.Requires
+
+class AndroidStudioIntegrationTest extends AbstractProfilerIntegrationTest {
+
+    @Requires({ StudioFinder.findStudioHome() })
+    def "benchmarks Android Studio sync with latest gradle version"() {
+        given:
+        File sandbox = File.createTempDir('android-studio-test', 'sandbox')
```

Comment:
Let's use JUnit's `TemporaryFolder` rule instead.

Reply:
The rule is already present in `AbstractProfilerIntegrationTest`, so you can just use `tmpDir.newFolder('sandbox')` or something like that.

Reply:
done